File: tests/openmp_examples/tasking/task_dep.13.f90
!$omp   parallel masked  num_threads(5)
!$omp     task
!$omp     end task
!$omp     task depend(out: a)
!$omp     end task
!$omp     task depend(out: d)
!$omp     end task
!$omp     task depend(inout: omp_all_memory)
!$omp     end task
!$omp     task
!$omp     end task
!$omp     task depend(in: a,d)
!$omp     end task
!$omp   end parallel masked

File: tests/openmp_examples/tasking/task_dep.13.c
#pragma omp parallel masked num_threads(5)
#pragma omp task
#pragma omp task depend(out: a)
#pragma omp task depend(out: d)
#pragma omp task depend(inout: omp_all_memory)
#pragma omp task
#pragma omp task depend(in: a,d)
